Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -43,6 +44,78 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * moveAction event handler, launch the universal discovery widget
+         * to choose a location to move the content
+         *
+         * @method _selectLocation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _selectLocation: function (e) {
+            this.fire('contentDiscover', {
+                config: {
+                    title: "Select the location you want to move your content into",
+                    contentDiscoveredHandler: Y.bind(this._moveContent, this),
+                },
+            });
+        },
+
+        /**
+         * Move the content to the selected location
+         *
+         * @method _moveContent
+         * @protected
+         * @param {EventFacade} e
+         */
+        _moveContent: function (e) {
+            var app = this.get('app'),
+                initialActiveView = app.get('activeView'),
+                parentLocationId = e.selection.location.get('id'),
+                locationId = this.get('location').get('id'),
+                that = this,
+                notificationSuffixe = parentLocationId + locationId;
+
+            that.fire('notify', {
+                notification: {
+                    text: 'The move operation of your content has begun',
+                    identifier: 'move-notification-' + notificationSuffixe,
+                    state: 'started',
+                    timeout: 5,
+                }
+            });
+
+            this.get('capi').getContentService().moveSubtree(locationId, parentLocationId, function (error, response) {
```

Comment:
you should add a `move` method to the Location model and use it here to write something like:
```js
this.get('location').move({api: this.get('capi')}, parentLocationId, function (error, response) {
   // ...
});
```